src/Vts/MonteCarlo/DataStructures/SimulationInputProvider.cs

```diff
@@ -1229,5 +1231,109 @@ public static SimulationInput FluorescenceEmissionAOfXAndYAndZSourceInfiniteCyli
             );
         }
         #endregion
+        
+        #region tissue bounded by voxel region  
+        /// <summary>
+        /// Bounded Cylinder Tissue
+        /// </summary>
+        /// <returns>An instance of the SimulationInput class</returns>
+        public static SimulationInput PointSourceBoundedTissue()
+        {
+            return new SimulationInput(
+                100,
+                "point_source_bounded_tissue",
+                new SimulationOptions(
+                    0, // random number generator seed, -1=random seed, 0=fixed seed
+                    RandomNumberGeneratorType.MersenneTwister,
+                    AbsorptionWeightingType.Discrete,
+                    PhaseFunctionType.HenyeyGreenstein,
+                    new List<DatabaseType>() { }, // databases to be written

```

Comment:
Since this is in the new code, I figured I would point it out. Empty initializer replace with List<DatabaseType>()

We can leave this now and fix it with the others later.